furniro-front-end/src/app/Commons/components/HomeBanner.tsx

```diff
@@ -0,0 +1,9 @@
+import { backgroundData } from "../data/HomeData";
+const HomeBanner = () => {
+  return (
+    <div>
+      <img className="w-full h-full" src="backgroundData" alt="" />
```

Comment:
Image cannot go without alt and in nextjs you should use the Image component from next/image for images

Reply:
Resolved